src/FileCacheManager.php

```diff
@@ -49,6 +50,7 @@ public function __construct($isEnabled, $cacheFile, array $rules)
     {
         $this->isEnabled = $isEnabled;
         $this->cacheFile = $cacheFile;
+        $this->cacheFileRealDirName = dirname(realpath($cacheFile));
```

Comment:
does `realpath` work (correctly) in `phar`'s? (I recall there were some issues with it)